tools/bumbershoot/idpassemble/idpassemble.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<tool id="idpassemble" name="idpAssemble" version="0.1.0">
+    <description>Merge IDPicker databases from single files into a merged database, and filters the result at PSM/spectrum/peptide/protein/gene levels.</description>
+    <requirements>
+        <requirement type="package" version="3.0.8789">bumbershoot</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Job Failed" />
+        <regex match="^Error:.*$" source="both" level="fatal" />
+    </stdio>
+    <command>
+        idpAssemble
+            -MaxFDRScore $max_fdr_score
+            -MinDistinctPeptides $filter_at_gene_level_condition.min_distinct_peptides
+            -MinSpectra $filter_at_gene_level_condition.min_spectra
+            -MinAdditionalPeptides $filter_at_gene_level_condition.min_additional_peptides
+            -MinSpectraPerDistinctMatch $min_spectra_per_distinct_match
+            -MinSpectraPerDistinctPeptide $min_spectra_per_distinct_peptide
+            -MaxProteinGroupsPerPeptide $max_protein_groups_per_peptide
+            #if $filter_at_gene_level_condition.filter_at_gene_level
+            -FilterAtGeneLevel 1
+            #end if
+            -SummarizeSources 1
+            #if len($input) > 1
+                -MergedOutputFilepath output
+                #for $i in $input
+                    '${i.file_name}'
+                #end for
+            #else
+                '${input}'
+                ;
+                cp '${input}' output
+            #end if
+    </command>
+    <inputs>
+        <param name="input" type="data" format="idpdb" label="Input File(s)" multiple="true"/>
+        <param name="max_fdr_score" type="float" label="Max FDR Score" value="0.05" />
+        <conditional name="filter_at_gene_level_condition">
+            <param name="filter_at_gene_level" type="boolean" truevalue="1" falsevalue="0" label="Filter at Gene Level" help="Apply filters at the gene level (i.e. 'min distinct peptides per gene group' instead of 'min distinct peptides per protein group')"/>
```

Comment:
Sorry, this one doesn't make sense as a section. The Advanced Options in myrimatch.xml does (and I'm changing it). This boolean option (rendered as a yes/no switch rather than a checkbox) changes the meaning and text of several parameters below it, but the help text of the button makes that clear. It doesn't add or drop any parameters like an advanced options section does.